src/core/pointcloud/qgseptdecoder.cpp

```diff
@@ -466,4 +478,24 @@ QgsPointCloudBlock *QgsEptDecoder::decompressLaz( const QString &filename,
          );
 }
 
+QgsPointCloudBlock *QgsEptDecoder::decompressLaz( const QByteArray &byteArrayData,
+    const QString &nodeStr,
+    const QgsPointCloudAttributeCollection &attributes,
+    const QgsPointCloudAttributeCollection &requestedAttributes )
+{
+  QString filename = QDir::tempPath() + QDir::separator() + QStringLiteral( "%1.laz" ).arg( nodeStr );
+  std::ofstream file( filename.toStdString(), std::ios::binary | std::ios::out );
```

Comment:
No luck with string stream? If laz-perf does not like that, can you please prepare a test code to demonstrate the issue (e.g. on pastebin / github gist)

Reply:
I was trying a different class previously but now using std::istringstream did work.